ui/raidboss/timeline.js

```diff
@@ -290,7 +304,7 @@ class Timeline {
       for (const trigger of triggers) {
         let found = false;
         for (const event of this.events) {
-          if (event.name.match(trigger.regex)) {
+          if (trigger.regex && trigger.regex.test(event.name)) {
```

Comment:
Thanks, I think this is better, because it means that anything missing a regex will throw an error.